NetTopologySuite.IO.GeoPackage/NetTopologySuite.IO.GeoPackage.csproj

```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="GeoAPI" Version="$(GeoAPIPackageReferenceVersion)" />
-    <PackageReference Include="NetTopologySuite" Version="1.15.2" />
+    <PackageReference Include="GeoAPI.Core" Version="1.7.5" />
```

Comment:
This ought to use `$(GeoAPIPackageReferenceVersion)` instead of hardcoded `1.7.5`

Reply:
yep, I'm fixing also in spatialite and test project